dataprofiler/tests/data_readers/test_csv_data.py

```diff
@@ -159,28 +161,23 @@ def setUpClass(cls):
                  count=9, delimiter=',', has_header=[3],
                  num_columns=3, encoding='utf-8'),            
         ]
-        cls.output_file_path = None
 
-    def test_is_match_for_string_streams(self):
-        """
-        Determine if the csv file can be automatically identified from
-        string stream
-        """
-        for input_file in self.input_file_names:
-            with open(input_file['path'], 'r',
-                      encoding=input_file['encoding']) as fp:
-                buffer = StringIO(fp.read())
-                self.assertTrue(CSVData.is_match(buffer))
+        cls.buffer_list = []
+        for input_file in cls.input_file_names:
+            buffer = StringIO(open(input_file['path'], 'r', encoding=input_file['encoding']).read())
+            buffer_info = input_file.copy()
+            buffer_info['path'] = buffer
+            cls.buffer_list.append(buffer_info)
+        
+        for input_file in cls.input_file_names:
+            buffer = BytesIO(open(input_file['path'], 'rb').read())
+            buffer_info = input_file.copy()
+            buffer_info['path'] = buffer
+            cls.buffer_list.append(buffer_info)
```

Comment:
could we slim this into a single list? also bc of the open statement, these aren't exactly getting closed:
```
for input_file in cls.input_file_names:
   # add StringIO
    buffer_info = input_file.copy()
    with open(input_file['path'], 'r', encoding=input_file['encoding']) as fp:
        buffer_info['path'] = StringIO(fp.read())
    cls.buffer_list.append(buffer_info)
    
    # add BytesIO
    buffer_info = input_file.copy()
    with open(input_file['path'], 'rb') as fp:
        buffer_info['path'] = BytesIO(fp.read())
    cls.buffer_list.append(buffer_info)
```